playbooks/roles/cyclecloud_cluster/projects/openpbs/cluster-init/scripts/2-installpbs.sh

```diff
@@ -12,8 +12,8 @@ fi
 
 echo "Configuring PBS"
 
-sed -i 's/CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME/scheduler/' /etc/pbs.conf
-sed -i 's/CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME/scheduler/' /var/spool/pbs/mom_priv/config
+sed -i 's/CHANGE_THIS_TO_PBS.*_SERVER_HOSTNAME/scheduler/' /etc/pbs.conf
```

Comment:
This is to address the change from PBS Pro to OpenPBS. Since v20 is OpenPBS I added this but there were compatibility issues and i decided to make v19 from source.

Reply:
it should work for both but we can remove for now

Reply:
reverted the change. this is separate change only for PBS version.